### PR TITLE
Datapackages in clojure tutorial

### PR DESCRIPTION
**Pull request by serahkiburu**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ install:
 - pip install lektor
 - pip install boto3
 - pip list
+- npm i -g npm
```
Comment (roll) [inline .travis.yml]: Not sure why)

**Review by roll — APPROVED** [review]
:+1:

File: .travis.yml (hunk 1 above)
Comment (serahkiburu) [reply .travis.yml]: added that to get around this issue https://travis-ci.org/frictionlessdata/website/builds/421934455#L602

Comment (serahkiburu) [reply .travis.yml]: but since removing `package-lock.json` fixed it, it's safe to leave it out.

**Pull request merged**